sdlf-ddk-lightweight/README.md

```diff
@@ -36,7 +46,7 @@ The Purpose of the DDK Serverless Data Lake Framework is to shorten the deployme
 To complete this deployment, you'll need the following in your local environment
 
 Programmatic access to an AWS Account
-Python (version 3.7 or above) and its package manager, pip (version 9.0.3 or above), are required
+Python (version 3.9 or above) and its package manager, pip (version 9.0.3 or above), are required
```

Comment:
Are we not able to support `3.7/8` anymore?

Reply:
@noah-paige I think you had suggested this change. I know `Protocol` is only supported in `3.8+`, but I don't remember/know what prompted you to suggest bumping to `3.9` minimum?

Reply:
I was running into TyperErrors when running `cdk synth` and Python 3.8.X in the past - a fix to make the example compatible with Python 3.8+ would be to not use dicts as parameterized generic types and instead use Dict (i.e. `dict[str, Any] --> Dict[str,Any]`)

and add `from typing import Dict ` at each file used

Reply:
@malachi-constant thoughts? I can make the change Noah proposed above to allow compatibility with 3.8 if you think it's worth it. Otherwise I will leave it as is.

Reply:
I personally think it's worth it.

Reply:
fixed.